### ISSUE TITLE
Showing error message even if the field type is string

### ISSUE THREAD

**Issue by IntegrityFoundation** [unclead/yii2-multiple-input#148]
I am using yii2-multiple-input widget for having multiple inputs for an attribute Taluka. While entering data, into field and clicking on submit, error is displayed as attribute must be a string even if the type of that attribute is string in database table.

**Comment by IntegrityFoundation** [comment]
View file (_form.php)

<?php

use yii\helpers\Html;
//use yii\widgets\ActiveForm;
use yii\bootstrap\ActiveForm;
use app\models\District;
use yii\helpers\ArrayHelper;
use wbraganca\dynamicform\DynamicFormWidget;
use unclead\multipleinput\MultipleInput;
use bupy7\dynafields\DynaFields;


/* @var $this yii\web\View */
/* @var $model app\models\Taluka */
/* @var $form yii\widgets\ActiveForm */
?>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
<script>

$(document).ready(function() {



});


</script>

<div class="taluka-form">

   <?php $form = ActiveForm::begin(['id' => 'dynamic-form', 'layout' => 'horizontal',
    'fieldConfig' => [
        'template' => "{label}\n{beginWrapper}\n{input}\n{hint}\n{error}\n{endWrapper}",
        'horizontalCssClasses' => [
           'label' => 'col-sm-5',
            //'offset' => 'col-sm-offset-2',
            //'wrapper' => 'col-sm-7',
            'error' => '',
            'hint' => '',

        ],
    ],]);?>

<div class="panel panel-primary " > 
<div class="panel panel-heading"><font size="3"><b>District</b></font></div>
<div class="row">
<div class="col-sm-4">

    <?= $form->field($model, 'DistrictId')->dropDownList(ArrayHelper::map(District::find()->all(),'DistrictId','District'), ['prompt' => 'Select District']) ?>

		
	<?= $form->field($model, "Taluka")->widget(MultipleInput::className(), [
    'max' => 4])?>
 
</div>


</div>
</div>
    <div class="form-group">
        <?= Html::submitButton($model->isNewRecord ? 'Create' : 'Update', ['class' => $model->isNewRecord ? 'btn btn-success' : 'btn btn-primary']) ?>
    </div>

    <?php ActiveForm::end(); ?>

</div>


Model:

<?php

namespace app\models;

use Yii;
class Taluka extends \yii\db\ActiveRecord
{
    /**
     * @inheritdoc
     */

	
    public static function tableName()
    {
        return 'taluka';
    }

    /**
     * @inheritdoc
     */
    public function rules()
    {
        return [
            [['DistrictId', 'Taluka'], 'required'],
            [['DistrictId'], 'integer'],
            [['Taluka'], 'string', 'max' => 100],
            [['DistrictId'], 'exist', 'skipOnError' => true, 'targetClass' => District::className(), 'targetAttribute' => ['DistrictId' => 'DistrictId']],
        ];
    }

    /**
     * @inheritdoc
     */
    public function attributeLabels()
    {
        return [
            'TalukaId' => 'Taluka ID',
            'DistrictId' => 'District',
            'Taluka' => 'Taluka',
        ];
    }
}

**Comment by unclead** [comment]
@IntegrityFoundation still actual?

**Comment by IntegrityFoundation** [comment]
It is working
